index.bs

```diff
@@ -1432,8 +1429,131 @@ following additional fields:
 Sessions {#streaming-sessions}
 ------------------------------------
 
-TODO
+To start a streaming session, a sender may send a
```

Comment:
What if the receiver wants to start a streaming session?

Reply:
See, e.g., Issue #176.

Reply:
The receiver wants to start sending?  It could take on the role of media sender and start a session in the other direction.

The receiver wants to start receiving?  It could send a "please start a streaming session with me" message, I guess.   We could do something more complex to save a round trip, but I don't think that's worth it (SDP offer/answer has way too much complexity to do this 1-round-trip offer-to-receive type stuff).  But assuming we add such a message "streaming-session-receive-request", what would the receiver say it wants to receive?  "Please send me your desktop?"  "Please send me a tab"?  What would the controller do?  Pop up a some UI to the user saying "a TV wants you to stream it something"?  We don't have anything like this in the Presentation and Remote Playback APIs, do we?

Reply:
This would be for allowing apps to request media streams from capture devices or remote renderers, for example from IP webcams, or maybe VR headsets.  Or an application on the receiver could generate a stream.

Maybe discuss later as part of the bidirectional case?  (This is a specific case of that.)

Reply:
In that case, I think a "please offer me what streaming you have" message would make sense.  We can add it in a separate PR.

Reply:
OK, I specified that.